recipes/PyAV/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "av" %}
```

Comment:
Any chance you will have time to do this today, @danielballan? Currently this naming issue is causing us problems converting other recipes to feedstocks. If you don't have time, I think we should remove it and would then need you to re-add the recipe fresh in a new PR. Thoughts?